tests/test_routes.py

```diff
@@ -176,3 +176,11 @@ def test_create_order_bad_status(self):
         test_order["status"] = "created"  # wrong value
         response = self.app.post(BASE_URL, json=test_order)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_list_orders(self):
+        """ It should list orders"""
+        self._create_orders(5)
+        response = self.app.get(BASE_URL)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
```

Comment:
Please move this test to the happy tests section above.

Reply:
done